project/plugins.sbt

```diff
@@ -6,5 +6,5 @@ addSbtPlugin("com.typesafe.play"  % "sbt-plugin"         % "2.8.20")
 addSbtPlugin("uk.gov.hmrc"        % "sbt-distributables" % "2.2.0")
```

Comment:
2.4.0

Reply:
Pls can you go from Scala 2.13.3 to 2.13.12 in AppDependencies & bootstrapVersion from 7.21 to 7.23, thx